icomoon/style.css

```diff
@@ -0,0 +1,60 @@
+@font-face {
+  font-family: 'icomoon';
+  src:  url('fonts/icomoon.eot?5jrnyf');
+  src:  url('fonts/icomoon.eot?5jrnyf#iefix') format('embedded-opentype'),
+    url('fonts/icomoon.ttf?5jrnyf') format('truetype'),
+    url('fonts/icomoon.woff?5jrnyf') format('woff'),
+    url('fonts/icomoon.svg?5jrnyf#icomoon') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'icomoon' !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-icon-case:before {
```

Comment:
Why do you need `.icon-icon-...`? Please, fix all the icon classes.
Maybe it will be better to recreate icomoon font.